.buildkite/run.sh

```diff
@@ -21,13 +21,23 @@ echo ""
 echo "--- ccache stats at start"
 ccache -s
 
-if [ -n "${BUILDKITE_PULL_REQUEST_BASE_BRANCH}" ]; then
-   ./scripts/ci/run_ci.sh  -c -b ${BUILDKITE_PULL_REQUEST_BASE_BRANCH} -r origin \
-	   -m ${JOB_NUM} -M ${BUILDKITE_PARALLEL_JOB_COUNT} -p ${BUILDKITE_PULL_REQUEST}
+if [ -n "${DAILY_BUILD}" ]; then
+   SANITYCHECK_OPTIONS=" --inline-logs -N --build-only --all --retry-failed 3 -v "
```

Comment:
Is the ```--build-only``` intentional? No 'running' of tests in a daily build?

Reply:
Yes, its what we've been doing for some time.

Reply:
everything that can be run is already run every time we push to mainline, so repeating this nightly does not make sense.